wiki/Skinning/Interface/en.md

```diff
@@ -3,93 +3,118 @@
 
 # Interface
 
-_See also: [Skinning Interface Tutorial](/wiki/Skinning_Interface_Tutorial) and [Interface](/wiki/Interface)_
+*Main page: [Skinning](/wiki/Skinning).*
 
-Interface skinning elements are used in multiple game modes or parts of the osu!client's user interface.
+*See also: [Skinning Interface Tutorial](/wiki/Skinning_Interface_Tutorial) and [Interface](/wiki/Interface)*
 
-## Main Menu
+Interface skinning elements are used in multiple game modes or parts of the osu! client's user interface.
 
-![](img/menu-background.jpg "menu-background.jpg")
+## Main menu
 
-| Animatable   | Beatmap Skinnable |
-|:------------:|:-----------------:|
-| ![No][false] | ![No][false]      |
+`menu-background.jpg`
 
-**Notes:**
+![](img/menu-background.jpg)
 
-- [osu!supporter](/wiki/osu!supporter) required
-- This element is used as the playfield if map does not have a background.
-- This element may use the `.jpg` format.
+| Versions | Animatable | Beatmap Skinnable | Blend Mode | Origin | Suggested SD Size |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| All | ![No][false] | ![No][false] | Normal | Centre | 1366x768 |
+
+Notes:
+
+- [osu!supporter](/wiki/osu!supporter) required.
+- By default, osu! has 5 background images that it will cycle through.
+  - If this element is skinned and the user has osu!supporter tag, this element will override this behaviour.
+- This element is used as the playfield if beatmap does not have a background.
+- This element uses the `.jpg` extension.
+  - `.png` images can be used, however, they must end with `.jpg` to work.
```

Comment:
> `.png` images can be used, however, they must end with `.jpg` to work.

This wording sounds like the file must be named `##.png.jpg`.

You better check with two different backgrounds, one regular `.png-file` and one regular `.jpg-file`, one at a time. If the `.png` doesn't work just state that only `.jpg` works but if both work you have to remove the `they must end with .jpg to work` part

Reply:
Here is what I found out:

`menu-background.jpg` notes:
- If seasonal backgrounds is set to `Always` the seasonal backgrounds will override this
- If seasonal backgrounds is set to `Sometimes` the seasonal backgrounds will override this while they are in active use *NOTE: (I assume this is the case)*
- `.jpg` extension only
  - If image type is `.png` change the extension to `.jpg`
    - If background is transparent, background will be black
- The player can drag and drop an image to **overwrite** the skin's skinned image